explainer.md

```diff
@@ -347,6 +348,39 @@ try {
   /* SPC cannot be used; merchant should fallback to traditional flows */
 }
 ```
+## User Experience in Failure Modes
+
+As the diagram above illustrates, the SPC "happy path" involves presentation of a transaction confirmation dialog followed by biometric authentication via the underlying platform.
+
+Below are different failure modes and the corresponding expected user experience.
+
+### No Available Authenticator
+
+When the credential IDs provided as input do not match an available authenticator, the browser:
+
+* displays a notice to the user, and
+* returns false to the caller.
+
+### User Cancels Transaction Confirmation
+
+When the user cancels the transaction confirmation dialog, the browser:
+
+* returns an error message ("NotAllowedError").
+
+### User Cancels Authentication Prompt
+
+When the user cancels the underlying platform prompt to authenticate, the browser:
+
+* returns an error message ("NotAllowedError").
+
+### User Fails to Authenticate
+
+When the user fails to authenticate, the underlying platform behavior will typically involve providing multiple opportunities to retry.
```

Comment:
Add that once/if the authenticator returns failure to the browser, the browser will return an error message to the caller?